akka-stream-tests/src/test/scala/akka/stream/scaladsl/TakeLastSinkSpec.scala

```diff
@@ -25,6 +25,36 @@ class TakeLastSinkSpec extends StreamSpec {
       result should be(Seq(4, 5, 6))
     }
 
+    "return top three student based on GPA correctly" in {
+      implicit val ex = system.dispatcher
+      //#takeLast-operator-example
+      case class Student(name: String, gpa: Double)
+
+      val students = List(Student("Alison", 4.7), Student("Adrian", 3.1), Student("Alexis", 4),
+        Student("Benita", 2.1), Student("Kendra", 4.2), Student("Jerrie", 4.3)).sortBy(_.gpa)
+
+      val sourceOfStudents = Source(students)
+
+      val result = sourceOfStudents.runWith(Sink.takeLast(3))
```

Comment:
Putting the explicit type here would be good for readers, so they really understand that they get a future back

Reply:
done.